x-pack/plugins/elastic_assistant/server/lib/langchain/graphs/default_assistant_graph/nodes/translations.ts

```diff
@@ -10,8 +10,10 @@ const YOU_ARE_A_HELPFUL_EXPERT_ASSISTANT =
 const IF_YOU_DONT_KNOW_THE_ANSWER = 'Do not answer questions unrelated to Elastic Security.';
 
 export const DEFAULT_SYSTEM_PROMPT = `${YOU_ARE_A_HELPFUL_EXPERT_ASSISTANT} ${IF_YOU_DONT_KNOW_THE_ANSWER}`;
-
-export const GEMINI_SYSTEM_PROMPT =
-  `ALWAYS use the provided tools, as they have access to the latest data and syntax.` +
-  "The final response is the only output the user sees and should be a complete answer to the user's question. Do not leave out important tool output. The final response should never be empty. Don't forget to use tools.";
+const ALLENS_PROMPT =
+  'You are an assistant that is an expert at using tools and Elastic Security, doing your best to use these tools to answer questions or follow instructions. It is very important to use tools to answer the question or follow the instructions rather than coming up with your own answer. Tool calls are good. Sometimes you may need to make several tool calls to accomplish the task or get an answer to the question that was asked. Use as many tool calls as necessary.';
+const KB_CATCH =
```

Comment:
I found if I do not add this catch and Gemini gets empty knowledge base results from a user question, it will not try to answer. See the differences is these traces for [before](https://smith.langchain.com/public/2c19a0dc-9298-49d7-8bf0-51fa397281ac/r) and [after](https://smith.langchain.com/public/d15e36f8-d1d6-466a-9ab1-7199a3fdc203/r) I added `KB_CATCH`